.gitignore

```diff
@@ -28,6 +28,14 @@ _testmain.go
 /out
 /_gopath
 
+deploy/addons/addon-manager.yaml                           
```

Comment:
This feels weird. It needs a comment, and a wildcard so that it doesn't have to be manually maintained every time a new addon is created.

Reply:
I added the comments now. I don't see any approach to have a wid card to ignore these files.